.gitignore

```diff
@@ -15,6 +15,7 @@ yarn-error.log*
 .vscode/
 public/
 
-subgraph.yaml
+*.yaml
```

Comment:
any specific reason for this? if it's to support the `filename` arg for the prepare.js script, i'll address that in other comments there

Reply:
I had a thing that let you specify a manifest for graft purposes, but it probably makes sense to get rid of it.